bin/templates/cordova/lib/build.js

```diff
@@ -530,8 +609,54 @@ module.exports.findBestApkForArchitecture = function(buildResults, arch) {
     throw new Error('Could not find apk architecture: ' + arch + ' build-type: ' + buildResults.buildType);
 };
 
+function PackageInfo(keystore, alias, keystorePassword, password, storeType) {
+    keystore = keystore.replace(/\\/g, '\\\\');
```

Comment:
This escaping will need to be done for all of the values I think. Would be more appropriate to do it within the toProperties function